### ISSUE TITLE
Error: Invalid example for `success` exit:

### ISSUE THREAD

**Issue by galcivar** [particlebanana/machinepack-stripe#5]
Hi! I have the following data:

```
subcription: 'sub_6caAxDfPt47Ec9'
stripe: 'cus_6cUrtcP72Q2urM'
```

Passed to a method as following:

```
  getSubscription: function(subscription, stripe, cb) {
    var stripeKey = sails.config.gedoc.stripeKey;
    var Stripe = sails.config.machinepack.Stripe;
    Stripe.retrieveSubscriptionDetails({
      apiKey: stripeKey,
      sub: subscription,
      customer: stripe,
    }).exec({
      error: function(err) {
        console.log("getSubscription err", err);
        cb(err,null);
      },
      success: function(SubscriptionDetails) {
        cb(null,SubscriptionDetails);
      },
    });
  }
```

This responds an Error with the following text:

```
[Error: Invalid example for `success` exit: { id: 'sub_6PefH3obLtjcCV',
  plan:
   { interval: 'year',
     name: 'New plan name',
     created: 1432558607,
     amount: 2000,
     currency: 'usd',
     id: 'rhohub-gold-year',
     object: 'plan',
     livemode: false,
     interval_count: 1,
     trial_period_days: null,
     metadata: {},
     statement_descriptor: null },
  object: 'subscription',
  start: 1434104747,
  status: 'active',
  customer: 'cus_6PdfAh2dIpK9G0',
  cancel_at_period_end: false,
  current_period_start: 1434104747,
  current_period_end: 1465727147,
  ended_at: null,
  trial_start: null,
  trial_end: null,
  canceled_at: null,
  quantity: 1,
  application_fee_percent: null,
  discount: null,
  tax_percent: null,
  metadata: {} }]
```

The ID for stripe and subscription are correct yet I get that weird response. Any ideas?

**Comment by galcivar** [comment]
Any news on this?

**Comment by lukeheath** [comment]
@galcivar - I ran into the same issue. The example success object needs to validate as JSON, which means quotes around all keys. I just copied Stripe's response object and fed it through a formatter (http://www.freeformatter.com/json-formatter.html) and that cleared up the error. Apparently this only happens if there is an object within your success object. If it's a single level object it doesn't have to validate as JSON. Not sure what's up with that.

![image](https://cloud.githubusercontent.com/assets/2495927/9961578/c8c43896-5de6-11e5-9a94-a021ca1a661e.png)
